get_no_of_revisions_for_title.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
```

Comment:
This example could be a bit more expanded. We could make it as such so that we show how to "get first five revisions of a page after a particular time either excluding or including edits made by a particular user."